### PR TITLE
Add HTML lists to export

### LINKED ISSUE #633: Export HTML lists
This is one part of #629. Current HTML export is as a table, but I would also like to support 1`--format html-list`. They'll look a bit like #631:

```
<ul>
  <li><em>CURIE</em> <a href="http://purl.obolibrary.org/obo/GO_0005622">GO:0005622</a></li>
  <li><em>LABEL</em> <span>intracellular</span></li>
  <li><em>SubClass Of</em> <span><a href="http://purl.obolibrary.org/obo/GO_0044464">cell part</a></span></li>
</ul>
<ul>
  <li><em>CURIE</em> <a href="http://purl.obolibrary.org/obo/GO_ 044424">GO:0044424 </a></li>
  <li><em>LABEL</em> <span>intracellular part</span></li>
  <li><em>SubClass Of</em> <span><a href="http://purl.obolibrary.org/obo/GO_0044464">cell part</a></span> | <span><a href="http://purl.obolibrary.org/obo/BFO_0000050">part of</a> some <a href="http://purl.obolibrary.org/obo/GO_0005622">intracellular</a></span></li>
</ul>
```

Maybe the values should be a sublist (styled as a Boostrap inline list)?

### PR DESCRIPTION
**Pull request by beckyjackson**
Resolves #633

This should probably wait on #691 and #699 before merging

- [x] `docs/` have been added/updated
- [ ] tests have been added/updated
- [x] `mvn verify` says all tests pass
- [ ] `mvn site` says all JavaDocs correct
- [x] `CHANGELOG.md` has been updated

Adds `html-list` format to `export` to output reports in a simple HTML list:

image

I tried using the Bootstrap inline lists, but it looked kind of weird. This will also support the `--standalone` option that we are adding in #691 - `--standalone false` will not include the `<head>` and `<body>` tags.

Also fixes a minor bug with the HTML renderer in which `&` and `<` symbols in text would cause issues with the HTML. These are now replaced by `&amp;` and `&lt;`.

### CODE REVIEW

File: robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java [hunk 1]
```diff
@@ -92,7 +92,8 @@ public void visit(OWLNamedIndividual individual) {
     write(
         String.format(
             "<a href=\"%s\">%s</a>",
-            individual.getIRI().toString(), getShortFormProvider().getShortForm(individual)));
+            individual.getIRI().toString(),
+            escape(getShortFormProvider().getShortForm(individual))));
```
Comment (jamesaoverton) [inline robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java:96]: Why is this one not escaped, when all the others are?

File: robot-core/src/main/java/org/obolibrary/robot/export/Row.java [hunk 2]
```diff
@@ -275,6 +275,42 @@ public String toHTML(List<Column> columns, String split) {
...
+    for (Column c : columns) {
+      String columnName = c.getDisplayName();
+      Cell cell = cells.getOrDefault(columnName, null);
+      if (cell != null) {
+        // Only add to HTML list if not null and values is not empty
+        List<String> values = cell.getDisplayValues();
+        if (values.size() == 0) {
+          continue;
+        }
+        sb.append("\t<li>\n");
```
Comment (jamesaoverton) [inline robot-core/src/main/java/org/obolibrary/robot/export/Row.java:294]: I'm not sure why we're using tabs for the HTML output, when we don't use tabs anywhere else in any of our projects (TSV doesn't count).

File: robot-core/src/main/java/org/obolibrary/robot/export/Table.java [hunk 3]
```diff
@@ -332,6 +332,20 @@ public String toHTML(String split, boolean standalone, boolean includeJS) {
     return sb.toString();
   }
 
+  public String toHTMLList() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("<head>\n")
+        .append(
+            "\t<link rel=\"stylesheet\" href=\"https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css\">\n")
```
Comment (jamesaoverton) [inline robot-core/src/main/java/org/obolibrary/robot/export/Table.java:339]: I'd like all these versioned links factored out to shared variables. This Bootstrap version doesn't match others in this same file.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
Sorry for the long delay.

Overall this looks good but I'd like some changes.

File: robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java (hunk 1 above)
Comment (beckyjackson) [reply robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java:96]: I think it is escaped? ` escape(getShortFormProvider().getShortForm(individual))`

File: robot-core/src/main/java/org/obolibrary/robot/export/Row.java (hunk 2 above)
Comment (beckyjackson) [reply robot-core/src/main/java/org/obolibrary/robot/export/Row.java:294]: I guess to make the HTML more legible. Should I take it out?

File: robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java (hunk 1 above)
Comment (jamesaoverton) [reply robot-core/src/main/java/org/obolibrary/robot/ManchesterOWLSyntaxObjectHTMLRenderer.java:96]: Ok

File: robot-core/src/main/java/org/obolibrary/robot/export/Row.java (hunk 2 above)
Comment (jamesaoverton) [reply robot-core/src/main/java/org/obolibrary/robot/export/Row.java:294]: I would like the code to use spaces for indentation, please. Two spaces for each level works well for HTML, which can get quite deep.

**Review by jamesaoverton — APPROVED** [review]
Thanks!

**Pull request merged**